build/ublue-os-just/ublue-os-just.spec

```diff
@@ -49,20 +53,32 @@ mkdir -p -m0755  %{buildroot}%{_bindir}
 install -Dm755 %{SOURCE9} %{buildroot}%{_bindir}/ujust
 install -Dm755 %{SOURCE10} %{buildroot}%{_bindir}/ugum
 
+# Add bash library for use in just
+mkdir -p -m0755 %{buildroot}%{_libdir}/ujust/
+install -Dm644 %{SOURCE12} %{buildroot}%{_libdir}/ujust
```

Comment:
Looks like `${_libdir}` varies by platform...  so I'm thinking maybe this didn't get tested. ;-)

My suggestion on testing is:
1. build the config container locally to some image like `ubconfig`
2. add the rpms from the config layer in a simple container built FROM a fedora toolbox image
3. install the RPM and test `ujust` configs, etc

Doing the above, I saw the following:
```
[root@e0cd6e40ecbd share]# ujust|head -n2
/tmp/justzB7Bbq/_default: line 6: /usr/lib/ujust/libformatting.sh: No such file or directory
/tmp/justzB7Bbq/_default: line 7: Urllink: command not found

Available commands:
[root@e0cd6e40ecbd share]# rpm -ql ublue-os-just 
/etc/profile.d/ublue-os-just.sh
/usr/bin/ugum
/usr/bin/ujust
/usr/lib64/ujust/libcolors.sh
/usr/lib64/ujust/libformatting.sh
/usr/lib64/ujust/libfunctions.sh
/usr/lib64/ujust/ujust.sh
/usr/share/ublue-os/just
/usr/share/ublue-os/just/00-default.just
/usr/share/ublue-os/just/10-update.just
/usr/share/ublue-os/just/20-clean.just
/usr/share/ublue-os/just/30-distrobox.just
/usr/share/ublue-os/just/40-nvidia.just
/usr/share/ublue-os/just/50-akmods.just
/usr/share/ublue-os/just/60-custom.just
/usr/share/ublue-os/just/70-nix.just
/usr/share/ublue-os/justfile
```

Suggestions on how to fix are either be smarter about the path in the source side (maybe complicated) or be more restrictive in the spec file `install`... I'd probably choose the latter.

I *think* that would change the target path for ujust lib stuff:
from: `%{buildroot}%{_libdir}/ujust`
to: `%{buildroot}/usr/lib/ujust` 

Please try it and see if you can reproduce what I saw and if the suggestion then fixes it.

Thanks!

Reply:
Ah yeah originally i used `/usr/libexec` and when i decided lib would be better, i looked up the macro for it in the fedora docs and did not notice it led to `/usr/%{_lib}`
and i agree being more restrictive in the spec file is the way to go.